### PR TITLE
added doc notes about vars plugins in precedence

### PR DESCRIPTION
**Pull request by bcoca**
##### SUMMARY

make note about where vars plugins fit in

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

docs

##### ANSIBLE VERSION

```
2.4/2.5
```

### CODE REVIEW

File: docs/docsite/rst/playbooks_variables.rst [hunk 1]
```diff
@@ -884,6 +884,7 @@ Basically, anything that goes into "role defaults" (the defaults folder inside t
 
 .. [1] Tasks in each role will see their own role's defaults. Tasks defined outside of a role will see the last role's defaults.
 .. [2] Variables defined in inventory file or provided by dynamic inventory.
+.. [3] This includes other vars pulled in by 'vars plugins', host_vars/group_vars are added by the default vars plugin shipped with Ansible.
```
Comment (dharmabumstead) [inline docs/docsite/rst/playbooks_variables.rst]: Please change the comma to a semicolon

**Review by dharmabumstead — CHANGES_REQUESTED** [review]
One small edit required. GTG after that.

File: docs/docsite/rst/playbooks_variables.rst (hunk 1 above)
Comment (bcoca) [reply docs/docsite/rst/playbooks_variables.rst]: that breaks the list and changes the meaning

**Comment by abadger** [comment]
@dharmabumstead ready for a new review.

**Review by dharmabumstead — APPROVED** [review]

**Review by dharmabumstead — APPROVED** [review]

**Comment by dharmabumstead** [comment]
Doc updates OK. Thanks @bcoca.

**Comment by abadger** [comment]
merged to devel and cherry-picked to stable-2.4 for 2.4.2beta2 release.

**Pull request merged**